index.js

```diff
@@ -1,105 +1,82 @@
 'use strict';
 
-var superagent = require('superagent'),
+var getJwt = require('frau-jwt'),
+	superagent = require('superagent'),
+	url = require('url'),
 	xsrf = require('frau-superagent-xsrf-token');
 
-var accessTokenExpiresAt = 0,
-	oauth2Enabled = true;
+function noop () {}
 
-function now() {
-	return Date.now()/1000 | 0;
+function isRelative/*ly safe*/ (url) {
+	return url.hostname === null;
 }
 
-function addHeaders(req) {
-	req = req.use(xsrf);
-	req.set('X-D2L-App-Id', 'deprecated');
-	return req;
+function endsWith (haystack, needle) {
+	var expectedPosition = haystack.length - needle.length;
+	var lastIndex = haystack.indexOf(needle, expectedPosition);
+	var result = lastIndex !== -1 && lastIndex === expectedPosition;
+	return result;
 }
 
-function processRefreshResponse(err, res) {
-	// Prior to OAuth 2 support the refreshcookie route returns 404.
-	if(res.status == 404) {
```

Comment:
Are we still expecting this to happen? Should I put this functionality into [frau-jwt](https://github.com/Brightspace/frau-jwt)?